hadoop/src/main/java/org/apache/carbondata/hadoop/api/CarbonFileInputFormat.java

```diff
@@ -162,7 +162,13 @@ public CarbonTable getOrCreateCarbonTable(Configuration configuration) throws IO
         // do block filtering and get split
         splits = getSplits(job, filter, externalTableSegments, null, partitionInfo, null);
       } else {
-        for (CarbonFile carbonFile : getAllCarbonDataFiles(carbonTable.getTablePath())) {
+        List<CarbonFile> carbonFiles = null;
```

Comment:
This change is not related to binary, suggest to keep it in separate PR

Reply:
ok, done